models/reviewDao.js

```diff
@@ -0,0 +1,72 @@
+const appDataSource = require("./dataSource");
+
+const createReview = async (userId, productId, comment) => {
+  try {
+    const result = await appDataSource.query(
+      `
+      INSERT INTO reviews(
+        user_id,
+        product_id,
+        comment
+      ) VALUES (?, ?, ?)
+      `,
+      [userId, productId, comment]
+    );
+    return result;
+  } catch (err) {
+    console.log(err);
+    const error = new Error("dataSource Error");
+    error.statusCode = 400;
+
+    throw error;
+  }
+};
+
+const deleteReview = async (userId, reviewId) => {
+  try {
+    const result = await appDataSource.query(
+      `
+    DELETE FROM reviews
+    WHERE user_id = ? AND id = ?
```

Comment:
```suggestion
    WHERE id = ?
```
이제는 특정 review의 ID를 받아서 지우는 작업이기 때문에 user_id 조건을 더할 필요가 없습니다.